web/src/main/java/org/cbioportal/web/util/ClinicalDataEnrichmentUtil.java

```diff
@@ -183,7 +183,17 @@ public ClinicalDataEnrichmentUtil(ClinicalDataService clinicalDataService) {
                                     sampleAttributes.stream().map(ClinicalAttribute::getAttrId)
                                             .collect(Collectors.toList()),
                                     ClinicalDataType.SAMPLE.name(), "SUMMARY")
-                            .stream().collect(Collectors.groupingBy(x -> x.getAttrId() + "SAMPLE",
+                            .stream()
+                            .filter(x -> { // filter are non numeric data to fix
+                                           // https://github.com/cBioPortal/cbioportal/issues/6228
+                                try {
```

Comment:
could you test for isNumeric?  i know in js, try/catch has performance implications.  this code is run a lot

Reply:
@jj are you ok with just filtering out the data this way?  not sure what else we could do.   we could just throw an error and have the feature fail.  we fixed frontend so it would cause total failure in app

Reply:
ok to filter out non-numerical data.

Reply:
Ideally we should give some information back to the user about this. Maybe do it as a follow up.